src/Microsoft.Restier.Core/ApiBuilder.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using Microsoft.Framework.DependencyInjection;
```

Comment:
@rayao I am not sure if there is any BIG gap between beta8, rc1 and even the latest version. If so, do we consider use rc1 or latest packages now to avoid any big change in the future?